lib/src/simple_op_store.rs

```diff
@@ -12,29 +12,185 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::collections::BTreeMap;
```

Comment:
To future-proof ourselves from switching the implementation of the `simple_op_store`, would we want to rename this to `thrift_op_store` instead?

Reply:
It's easy to rename the type later. What kind of future-proofing are you thinking about? Make us compatible with future out-of-tree API users? For that, we first need to spend some time to clean up the exposed API :) I haven't spent much time on that yet.

Reply:
Nothing major, just that if we change the implementation yet again, we'd have to repeat this process of "rename `simple_op_store` to `thrift_op_store`", "create new `simple_op_store`". Feel free to resolve this :)

Reply:
I think it's more likely that we'll add `advanced_op_store` (with a better name) with a more specialized storage format (not a file per object). I hope we won't have a reason to switch back to protobuf :)